### ISSUE TITLE
FutureWarning: The series.append method is deprecated. Use pandas.concat instead.

### ISSUE THREAD

**Issue by hyxxsfwy** [twopirllc/pandas-ta#531]
**Which version are you running? The lastest version is on Github. Pip is for major releases.**
```python
import pandas_ta as ta
print(ta.version)
```
- 0.3.14b0

**Do you have _TA Lib_ also installed in your environment?**
- Nope

**Did you upgrade? Did the upgrade resolve the issue?**
- Yes
- Nope

**Describe the bug**
When calling ta.mcgd(), `FutureWarning` reported.
```
FutureWarning: The series.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.
```

**To Reproduce**
`y_smooth = ta.mcgd(series, ma_len)`

**Expected behavior**
Use pandas.concat instead of series.append method when caculating mcgd().

Thanks!

**Comment by twopirllc** [comment]
@hyxxsfwy,

Yes. This has been known since Issue #340. It _may_ be time to "try" to update the place in ```core.py``` with ```concat```.

Also, you should update to the _development_ branch for [reasons](https://github.com/twopirllc/pandas-ta/tree/development)...

```sh
$ pip install -U git+https://github.com/twopirllc/pandas-ta.git@development
```

Kind Regards,
KJ

**Comment by FedericoCaceres** [comment]
not work for pythonv3.12. Coud u update it?
